src/glossary.rst

```diff
@@ -67,50 +67,39 @@ Glossary
       Submit number also appears in the job log path so that job log files
       don't get overwritten.
 
-
-   active
    active task
-      An active task is a task which is near ready to run, in the process of
-      running, or which requires user intervention. These are all the tasks
-      being actively managed by the scheduler at this point in the run. 
+      Tasks that are in the ``submitted`` and ``running`` states, i.e., those
+      with active jobs. Be aware that the :term:`active window` of the workflow
+      may also contain some tasks that are not active in this sense.
 
-      Active tasks are:
+   window
+   active window
+      The active window of a workflow contains only the tasks that Cylc needs
+      to actively manage in order to perform its job of scheduling efficiently:
 
-      - Tasks which have some, but not all of their
-        :term:`prerequisites <prerequisite>` satisfied.
-      - ``waiting`` tasks, that are actively waiting on:
+      - :term:`active tasks <active task>` (``submitted`` and ``running``, with active jobs)
+      - ``preparing`` tasks - i.e. tasks in the process of submitting jobs
+      - ``waiting`` tasks that are nearly ready to run but:
 
-        - :term:`xtriggers <xtrigger>`.
-        - :ref:`internal queues <InternalQueues>`
-        - :ref:`runahead limit <RunaheadLimit>`
+        - still have some (but not all) unsatisfied :term:`prerequisites <prerequisite>`
+        - are waiting on :term:`xtriggers <xtrigger>`,
+          :ref:`internal queues <InternalQueues>`, or the :ref:`runahead limit <RunaheadLimit>`
 
-      - ``preparing`` tasks - i.e. tasks in the process of submitting jobs
-      - ``submitted`` and ``running`` tasks - i.e. those with active jobs
       - tasks that reached a :term:`final status` without completing their
         :term:`required outputs <required output>`
         (e.g. a task failed where success was required).
 
-      Active tasks are in the ``n=0`` :term:`window <n-window>` which means they
-      will always be displayed in the GUI and Tui.
-
-      The distinction between active and non-active tasks is important for
-      the computing of the :term:`runahead limit`.
-
+   n-window
+      The GUI provides a view extending ``n`` (default ``n=1``)
+      graph edges out from the :term:`active window` of the workflow.
+      Thus in the context of the GUI the active window may also be
+      referred to as the `n=0` window.
```

Comment:
```suggestion
      By default the GUI provides a view extending ``n`` (default ``n=1``)
      graph edges out from the :term:`active window` of the workflow.
      In the context of the GUI the active window may also be
      referred to as the `n=0` window.
```

Reply:
I'll skip this suggestion, because:
 1. *"By default the GUI ... extending n (default n=1) edges"*: too much "default"!
 2. The first sentence provides the reason for the second, hence they should be connected by "Thus" or "So".